johnsonandjohnson/Bodiless-JS
# fix(migration_tool): Migration tool fix default value for disableTailwind

## Changes
- Fix logic for the default value of the `disableTailwind`.

## Test Instructions
- Run `npm run sites:clone-local starter`
- Run `npm run sites:update starter`
- Go to `cd sites/starter`
- Make sure `BODILESS_TAILWIND_THEME_ENABLED` is commented out in `.env.site` by default
- Run `npm run migrate`
- Make sure `BODILESS_TAILWIND_THEME_ENABLED` is now set to `0` and not commented out in `.env.site`.

- Comment `BODILESS_TAILWIND_THEME_ENABLED` in `.env.site` by adding `#` as a first char on that line.
- Add `"disableTailwind": false` to the `sites/starter/migration-settings.json`
- Run `npm run migrate`
- Make sure `BODILESS_TAILWIND_THEME_ENABLED` did not get updated.

## Related Issues
Closes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/124
